cas-server-support-redis-ticket-registry/src/main/java/org/apereo/cas/ticket/registry/TicketRedisTemplate.java

```diff
@@ -0,0 +1,29 @@
+package org.apereo.cas.ticket.registry;
+
+import org.apereo.cas.ticket.Ticket;
+import org.springframework.data.redis.connection.RedisConnectionFactory;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.data.redis.serializer.JdkSerializationRedisSerializer;
+import org.springframework.data.redis.serializer.RedisSerializer;
+import org.springframework.data.redis.serializer.StringRedisSerializer;
+
+/**
+ * @author serv
+ */
+public class TicketRedisTemplate extends RedisTemplate<String, Ticket> {
```

Comment:
Good. Now go through the rest of the files, account for changes and fix build failures please.